docs/site/Route.md

```diff
@@ -201,87 +201,7 @@ class MySequence extends DefaultSequence {
 }
 ```
 
-## Implementing HTTP redirects
+## Customizing routes
```

Comment:
Since the links below are to How-to guides, I am proposing to call this section to something like "Related guides" or "How-to guides".

```suggestion
## Related guides
```